docs/design/snapshotters.md

```diff
@@ -0,0 +1,32 @@
+# Snapshotters Config Option
+
+## Issue
+
+[#506](https://github.com/runfinch/finch/pull/506)
+
+## Supported snapshotters
+
+- [SOCI](https://github.com/awslabs/soci-snapshotter)
+- OverlayFS (default snapshotter of nerdctl/containerd)
+
+## ```snapshotters``` option
+
+Users can use the snapshotters option at ```${HOME}/.finch/finch.yaml``` to set the snapshotters they would like to use.
+All snapshotters listed will be installed on the VM, and the first snapshotter listed will be made the default snapshotter.
+The default snapshotter will be used whenever a snapshotter is needed and not specified for a command, and any other listed snapshotters can be used
+by specifying them with the ```--snapshotter``` flag (i.e. ```finch pull ...``` will use the default snapshotter and
+```finch --snapshotter={exampleSnapshotter} pull``` will use the snapshotter that is specified)
```

Comment:
nit: suggest putting this into 2 separate sentences
```suggestion
The default snapshotter will be used whenever a snapshotter is needed and not specified for a command.
Any other listed snapshotters can be used by specifying them with the ```--snapshotter``` flag:
```
and then show example as 
 ```bash
$ finch pull ... # use the default snapshotter
$ finch --snapshotter=mySnapshotter pull ... # use mySnapshotter
```